proposed/2021/BulkRestoreCoordination.md

```diff
@@ -127,11 +138,33 @@ The proposal is that each time NuGet determines a restore might need run, it che
     }
 ```
 
+#### Implementation details
+
+- Given that there are risks with this proposal, the NuGet implementation will have a means of disabling this feature.
+- NuGet will only account for projects that have registered an IVsProjectRestoreInfoSource. A PackageReference based project does not have to register an IVsProjectRestoreInfoSource.

```

Comment:
A PackageReference-based project is not required to register an IVsProjectRestoreInfoSource; however, not registering means that the project is responsible for restoring itself.